webpack.config.js

```diff
@@ -0,0 +1,75 @@
+const Encore = require('@symfony/webpack-encore');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/bfoujols/relearun/pullRequest?prid=8781771)
```suggestion
const Encore = require("@symfony/webpack-encore");
```